google/cloud/pubsub/internal/tracing_exactly_once_ack_handler_test.cc

```diff
@@ -55,10 +55,8 @@ MakeTestExactlyOnceAckHandler(
   EXPECT_CALL(*mock, delivery_attempt()).WillRepeatedly(Return(42));
   EXPECT_CALL(*mock, ack_id()).WillRepeatedly(Return(kTestAckId));
   EXPECT_CALL(*mock, subscription()).WillRepeatedly(Return(kTestSubscription));
-  auto span =
-      internal::GetTracer(Options{})->StartSpan("test-subscription subscribe");
   Span span_holder;
-  span_holder.SetSpan(span);
+  span_holder.span = internal::MakeSpan("test-subscription subscribe");
   return MakeTracingExactlyOnceAckHandler(std::move(mock), span_holder);
```

Comment:
optional nit (purely style): I would skip naming a `pubsub_internal::Span`:

```
auto span = internal::MakeSpan("test-subscription subscribe");
return MakeTracingExactlyOnceAckHandler(std::move(mock), {std::move(span)});
```

Someone like @scotthart would skip naming the otel span too:

```
return MakeTracingExactlyOnceAckHandler(
    std::move(mock), {internal::MakeSpan("test-subscription subscribe")});
```

Reply:
done